pkg/blob/blob.go

```diff
@@ -118,7 +120,7 @@ const (
 )
 
 var (
-	supportedProtocolList = []string{fuse, nfs}
+	supportedProtocolList = []string{fuse, fuse2, nfs}
```

Comment:
I am not sure about one thing here. Blobfuse (v1) needs "fuse (libfuse)" to be installed on the system. Is that done automatically by CSI driver or user does it manually, If its done automatically then we need to install "fuse3" on the system as well for v2 to work. On systems where "fuse3" is not supported, v2 can work with fuse2 however for that we have a seperate binary.

Reply:
Ubuntu OS manages those dependencies